CommonLibSF/include/RE/T/TESObjectCELL.h

```diff
@@ -12,7 +13,7 @@ namespace RE
 	public:
 		SF_RTTI_VTABLE(TESObjectCELL);
 		SF_FORMTYPE(CELL);
-
-		//
+		uint8_t        pad[0xD8];
```

Comment:
`std::uint8_t` for consistency, all `std` namespace members are written out fully qualified in CommonLib